### ISSUE TITLE
Template overrides

### ISSUE THREAD

**Issue by shakethatweight-simon** [alicolville/Weight-Tracker#557]
I can't find any template/layout files that can be overridden at the theme level?
https://www.ibenic.com/include-or-override-wordpress-templates/

For example we need to show the menu with words for accessibility reasons. Icons with no alt text are bad practice, but there appears to be no way to override this in the theme?

**Comment by alicolville** [comment]
I'm afraid you can't override them at theme level - only with CSS.
